src/task/src/metrics/inputs.ts

```diff
@@ -92,6 +93,17 @@ export default class Inputs {
     return this._fileMatchingPatterns;
   }
 
+  /**
+   * Gets the test matching patterns input, which is the set of globs specifying the files and folders to consider tests.
+   * @returns The test matching patterns input.
+   */
+  public get testMatchingPatterns(): string[] {
+    this._logger.logDebug("* Inputs.testMatchingPatterns");
```

Comment:
This log seems redundant. Is it supposed to be left here?

Reply:
I see it aligns with other logging in this file.
Suggest keeping the log message format uniform:
```suggestion
    this._logger.logDebug("* Inputs.testMatchingPatterns()");
```

Reply:
Yes, it's deliberate. I have it here for failure cases, where the logs get replayed. Adding these makes failure much easier to track back to a specific call site.

For the getters, the format is without the brackets, as this reflects how these getters are called.